packages/spartan/extra/gen_resolvconf.py

```diff
@@ -51,6 +51,7 @@ def check_server(addr):
 
 options timeout:1
 options attempts:3
+options edns0
```

Comment:
I think it's pretty reasonable to expect people's resolvers to support edns0 in this day.